packages/CalloutParagraph/CalloutParagraph.jsx

```diff
@@ -59,17 +61,45 @@ const TextWrapper = styled.div`
       padding-left: ${props => paddingValue.mobile[props.spacing]};
       padding-right: ${props => paddingValue.mobile[props.spacing]};
     }
+    > i {
```

Comment:
We will soon be dropping support for font icons and this css will not be needed. We can keep it for now, just something to note.